src/main/java/com/global/api/builders/ManagementBuilder.java

```diff
@@ -50,11 +50,13 @@
     private String transportData;
     private BigDecimal totalCredits;
     private BigDecimal totalDebits;
+    private Integer estimatedNumTxn;
+    private boolean isFinal;
```

Comment:
Not clear what the value is for. Name should be updated to reflect that this is for the last capture against an authorization.